lms/static/js/student_account/views/LoginView.js

```diff
@@ -19,12 +21,15 @@
                 formType: 'login',
                 requiredStr: '',
                 submitButton: '.js-login',
+                formSuccessTpl: formSuccessTpl,
+                formStatusTpl: formStatusTpl,
+                defaultFormErrorsTitle: gettext("We couldn't sign you in."),
```

Comment:
That makes sense, and I adjusted most of the Strings in this PR to use single quotes. However, I went with single quotes here because of the apostrophe. Is that ok, or are single quotes with an escape character preferred?